### ISSUE TITLE
Unable to use re-size attribute with Mat Expansion

### ISSUE THREAD

**Issue by developer707** [angular/components#16241]
#### Reproduction

https://stackblitz.com/edit/angular-kywskk?file=styles.css
Steps to reproduce:
1.  re-size mat expansion vertically.
2.  Try to collapse
 
 
#### Expected Behavior

What behavior were you expecting to see?
Mat expansion should close properly

#### Actual Behavior

What behavior did you actually see?
the content closes but blank background still appearing

#### Environment

- Angular:
- CDK/Material:
- Browser(s):
- Operating System (e.g. Windows, macOS, Ubuntu):

**Comment by josephperrott** [comment]
Closing as this is working as expected, this functionality is not something that is supported by the expansion panel.

This does not work as you expected in your stackblitz because the element being resized is `<mat-expansion-panel>`, not the expanding area itself (`mat-expansion-panel-body`).  I created this [fork of your stackblitz](https://stackblitz.com/edit/angular-kywskk-gaxmez?file=styles.css) which allows for resizing the content while maintaining the expansion panel ability.
